vnext/PropertySheets/External/Microsoft.ReactNative.Composition.CppApp.targets

```diff
@@ -13,7 +13,7 @@
 
   <ItemGroup>
     <PackageReference Include="boost" Version="1.76.0.0" />
-    <PackageReference Include="Microsoft.Toolkit.Win32.UI.XamlApplication" Version="6.1.3" />
+    <PackageReference Include="Microsoft.Toolkit.Win32.UI.XamlApplication" Version="6.1.3" Condition="'$(UseWinUI3)'!='true'"/>
```

Comment:
general feedback, we should not use this package, it's not maintained. See https://github.com/asklar/xaml-islands